sosialhjelp-common-api/src/main/kotlin/no/nav/sosialhjelp/api/fiks/FiksDigisosApi.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.sosialhjelp.api.fiks
+
+data class DigisosSak(
+        val fiksDigisosId: String,
+        val sokerFnr: String,
+        val fiksOrgId: String,
+        val kommunenummer: String,
+        val sistEndret: Long,
+        val originalSoknadNAV: OriginalSoknadNAV?,
+        val ettersendtInfoNAV: EttersendtInfoNAV?,
+        val digisosSoker: DigisosSoker?,
+        val tilleggsinformasjon: Tilleggsinformasjon?
+)
+
+data class OriginalSoknadNAV(
+        val navEksternRefId: String,
+        val metadata: String,
+        val vedleggMetadata: String,
+        val soknadDokument: DokumentInfo,
+        val vedlegg: List<DokumentInfo>,
+        val timestampSendt: Long
+)
+
+data class DokumentInfo(
+        val filnavn: String,
+        val dokumentlagerDokumentId: String,
+        val storrelse: Long
+)
+
+data class EttersendtInfoNAV(
+        val ettersendelser: List<Ettersendelse>
+)
+
+data class Ettersendelse(
+        val navEksternRefId: String,
+        val vedleggMetadata: String,
+        val vedlegg: List<DokumentInfo>,
+        val timestampSendt: Long
+)
+
+data class DigisosSoker(
+        val metadata: String,
+        val dokumenter: List<DokumentInfo>,
+        val timestampSistOppdatert: Long
+)
+
+data class Tilleggsinformasjon(
+        val enhetsnummer: String?
+)
+
+data class KommuneInfo(
+        val kommunenummer: String,
+        val kanMottaSoknader: Boolean,
+        val kanOppdatereStatus: Boolean,
+        val harMidlertidigDeaktivertMottak: Boolean,
+        val harMidlertidigDeaktivertOppdateringer: Boolean,
+        val kontaktPersoner: Kontaktpersoner?
```

Comment:
KS har lagt til to nye felter som ser ut som mangler her. 
Det er NKStilgang og behandlingsansvalig.